spring/spring-data/src/main/java/io/quarkus/ts/spring/data/di/ListInjectionBean.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.ts.spring.data.di;
+
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import io.quarkus.arc.All;
+import io.quarkus.arc.properties.IfBuildProperty;
+
+@ApplicationScoped
+@IfBuildProperty(name = ListInjectionBean.LIST_INJECTION_BEAN_ENABLED, stringValue = ListInjectionBean.LIST_INJECTION_BEAN_ENABLED_VALUE)
```

Comment:
Add comment above to explain why is this annotation used, e.g. "not to interfere in other tests"